hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -147,13 +197,20 @@ Network boot functionality is included in the 2020-02-13 Raspbian Buster release
 sudo apt update
 sudo apt upgrade
                                                                           
-# Select the stable bootloader release series. The default is 'critical' which is only updated 
-# for major bugs, security fixes or hardware compatibility changes.
-# Network boot is not enabled in the default bootloader.
+# Network boot is not yet enabled in the production bootloader image.
```

Comment:
Would changing this to "# Network boot is not yet enabled in the production (critical) bootloader image." add clarity or just confusion?

Reply:
changed text to state where network boot IS available rather than stating where it is not supported